samples/net/wifi/shell/overlay-p2p.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_WIFI_NM_WPA_SUPPLICANT_P2P=y
+CONFIG_WPA_CLI=y
+CONFIG_NRF70_P2P_MODE=y
```

Comment:
this doesn't belong here and should be auto-enabled by CONFIG_WIFI_NM_WPA_SUPPLICANT_P2P